apps/catalogue/src/components/detailView/RefFieldValue.vue

```diff
@@ -24,6 +28,20 @@ export default {
       required: true,
     },
   },
+  computed: {
+    convertToRouterParams() {
```

Comment:
this name sounds like a function, not a computed. I think just 'routerParams' would be fine here

Reply:
indeed.